src/main/scala/org/apache/mesos/chronos/scheduler/api/JobManagementResource.scala

```diff
@@ -84,6 +84,36 @@ class JobManagementResource @Inject()(val jobScheduler: JobScheduler,
                   case _ =>
                 }
             }
+          case t: TriggeredJob =>
```

Comment:
At some point two job types become semantically different enough that they should be separated. I agree, with this pull request on its own, I may as well just schedule a job for next the next century. However, if the ability to schedule multiple instances of the same job gets added (see the other comment) I think we should probably have another job type.